src/app/public/src/modules/edit-button/edit-button.component.html

```diff
@@ -0,0 +1,7 @@
+<button
```

Comment:
Mind wrapping this in a `<div class="stache-edit-button">` to make things easier to style in the future (or from another component)? If you don't like the DIV, adding a special class to the button would be fine, too: `stache-btn-edit`.